drivers/timer/rv32m1_lptmr_timer.c

```diff
@@ -136,6 +136,11 @@ uint32_t sys_clock_cycle_get_32(void)
 	return cycle_count + SYSTEM_TIMER_INSTANCE->CNR;
 }
 
+uint64_t sys_clock_cycle_get_64(void)
+{
+	return 0;
```

Comment:
@mbolivar-nordic - would be great to get your input here. It seems there is no 64-bit timer register on this platform (i.e. the lptimer).

For software emulation, I would like to increment a 32-bit "top half" each time a 32-bit cycle counter overflows (assuming a 1:1 ratio), shift and then OR it with the regular 32-bit value.

In this driver, where is the best location to increment the 32-bit "top half"?